cves/kernel/CVE-2016-8650.yml

```diff
@@ -335,9 +350,11 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: true
+  stacktrace_with_fix: true
+  note: |
+    there is a stack trace of the vulnerability's  pointer issue in the
+    commit that fixes the issue
```

Comment:
Should add this link which contains the stack trace (https://seclists.org/fulldisclosure/2016/Nov/76) or copy/paste it here